<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StatsWindow</class>
 <widget class="QWidget" name="StatsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Estadísticas de Zona de Interés</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="btnMedia">
     <property name="text">
      <string>📊 Calcular media</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnMediana">
     <property name="text">
      <string>📌 Calcular mediana</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnDesviacion">
     <property name="text">
      <string>📐 Calcular desviación estándar</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnMinMax">
     <property name="text">
      <string>🔽 Mínimo y máximo</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnArea">
     <property name="text">
      <string>📏 Calcular área</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnOutliers">
     <property name="text">
      <string>🚨 Detectar outliers</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnBoxplot">
     <property name="text">
      <string>📈 Mostrar boxplot</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnGuardarCSV">
     <property name="text">
      <string>💾 Guardar estadísticas</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelResultado">
     <property name="text">
      <string/>
     </property>
     <property name="minimumHeight">
      <number>24</number>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCustomPlot" name="customPlot">
     <property name="minimumHeight">
      <number>250</number>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
